GuildedRoseLLC/Items/StaticItemRepository.swift

```diff
@@ -4,20 +4,8 @@ public class StaticItemRepository : ItemRepository {
     public init() {}
     
     public func getItems(onSuccess: @escaping (_:[Item]) -> Void){
-        var items: [Item] = []
+        let items: [Item] = Item.testData
         
-        let args: String = CommandLine.arguments.count > 1 ? CommandLine.arguments[1] : ""
```

Comment:
Nice! It is always great to see a lot of deleted code.

Reply:
👍